samples/prometheus-grafana/sample.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Capture and visualize metrics using Prometheus and Grafana
+component: Metrics
+---
+
+[Prometheus](https://prometheus.io) is a monitoring solution for storing time series data like metrics. [Grafana](https://grafana.com) allows to query and visualize the data stored in Prometheus (and other sources). This sample demonstrates how to capture NServiceBus metrics, storing these in Prometheus and visualizing these metrics using Grafana.
+
+
+
+![Grafana NServiceBus processing time](grafana-graph.png)
+
+
+
+This sample reports the following metrics to Prometheus:
+
+ * Fetched messages per second 
+ * Failed messages per second
+ * Successful messages per second
+ * Critical time in seconds
+ * Processing time seconds
+
+
+
+ The sample also shows data like `Average message processing time in milliseconds` and `Maximum critical time in milliseconds`. For a detailed explanation of these metrics refer to the [metrics captured section in the metrics documentation](/nservicebus/operations/metrics.md#metrics-captured).
```

Comment:
> The sample also shows data like `Average message processing time in milliseconds` and `Maximum critical time in milliseconds`.

"like" feels awkward. there is not really a way of thinking of similar metrics in this context. perhaps just list all?